doc/source/report/path.rst

```diff
@@ -75,81 +57,78 @@ As we begin our analysis, we would like to explore the following points:
-random forest, SVM, gradient boosting. Each method should have following
-steps: - Cross validation: divide the data into train, validation, and
-test set - Tune the parameters: Based on the train and validation set,
-tune the parameter to maximize some pre-defined measurement. - Fit on
-the test set: Evaluate the performance of the classification on the test
-set.
-
-**Step 4**: Interpretation
-
-Hopefully we might employ the different model like logistic regression
-to get a sense of the effect size of each features on the response
-variables.
+We define a path based on a specific time threshold between movements. In more
+detail, we created a function that considers that time difference. If the time
+between movements exceeded the threshold, we considered a new path to have
+begun. That path was considered to have ended once the threshold had been
+exceeded again. Based on the advice of the Tecott lab, the default threshold
+was one second.
+
+**Step 2**: Clean Data
+
+Based on our definition of a path, we realized that we needed to clean the
+data to remove obvious outliers and noise. For instance, we found that the
+sensor platform generated extreme outliers such as an acceleration of -4000
+centimeters per second. In order to remove this noise, we created two
+functions: one to filter the paths and one to remove duplicate rows. The
+filter paths function uses a minimum number of points in a path to filter out
+paths that create noise. For example, a path that is only three points could
+simply be a mouse shifting weight from one foot to the other and back again.
+Since this is not the type of path we are interested in analyzing, it makes
+sense to filter them out. The remove duplicates function deals with an issue
+in the data where the same x and y coordinates appear in adjacent rows but
+with different timestamps. This caused a problem when computing the angles of
+the paths so, while it is a trivial removal, it is necessary.
+
+**Step 3**: Choose Key Features
+
+The path features we wrapped up into functions are as follows: 
+    - Path Length: Total distance covered 
+    - Path Speed: Average speed 
+    - Acceleration: Ratio of speed to the time difference at each point 
+    - Angles: Angle between adjacent vectors
+    - Radius of Path: Distance between the center point and every other point 
+    - Center Angles: Angle created by two adjacent radius vectors
+    - Area Covered: Area covered by the path, computed with center angle and
+      radii
+    - Area of Rectangle: Area of rectangle that contains the entire path
+    - Absolute Distance: Distance between the first and last points 
+
+**Step 4**:  Interpretation
+
+Below are the plots generated based on the features we calculated above, per
+strain, mouse, and day. With these plots, we can draw some initial conclusions
+about differences between strains in the results section.
+
+Results
+-------------------------
+
+.. figure:: figure/dist_path.png
```

Comment:
We should comment on these plots. For example:
"Strain 0 appeared to be the most active overall. For all mice in this strain, activeness peaked around days 6-8 and fell after that. In contrast, there is a huge variation in the activeness among mice of strain 1."